src/Microsoft.Sbom.Contracts/Contracts/Enums/ComplianceStandard.cs

```diff
@@ -3,14 +3,13 @@
 
 using System.Text.Json.Serialization;
 
-namespace Microsoft.Sbom.Parsers.Spdx30SbomParser.Entities.Enums;
+namespace Microsoft.Sbom.Contracts.Enums;
 
 /// <summary>
 /// Defines the different supported compliance standards.
 /// </summary>
 [JsonConverter(typeof(JsonStringEnumConverter))]
 public enum ComplianceStandard
 {
-    NTIA,
-    None,
+    NTIA
```

Comment:
Enums will always default to 0, so having `None` as the 0 value would be a best practice here

Reply:
It wasn't being used currently so I removed it, but it might be of use later as you mentioned so I will add it back.